packages/suite/src/views/guide/GuideArticle.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect, useState } from 'react';
+import ReactMarkdown from 'react-markdown';
+import * as guideActions from '@suite-actions/guideActions';
+import { useActions, useSelector } from '@suite-hooks';
+import styled from 'styled-components';
+import { variables } from '@trezor/components';
+import { Header, Content, ViewWrapper } from '@guide-components';
+
+const StyledMarkdown = styled.div`
+    color: ${props => props.theme.TYPE_LIGHT_GREY};
+    font-size: ${variables.FONT_SIZE.SMALL};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    line-height: 1.5;
+    padding: 0 0 32px 0;
+    h1,
+    h2,
+    h3,
+    h4,
+    h5,
+    h6 {
+        color: ${props => props.theme.TYPE_DARK_GREY};
+        font-weight: ${variables.FONT_WEIGHT.DEMI_BOLD};
+    }
+    h1 {
+        margin: 8px 0 16px;
+        font-size: ${variables.FONT_SIZE.BIG};
+    }
+    h2 {
+        margin-bottom: 8px 0 12px;
+        font-size: ${variables.FONT_SIZE.NORMAL};
+    }
+    h3,
+    h4,
+    h5,
+    h6 {
+        margin: 4px 0 12px;
+        font-size: ${variables.FONT_SIZE.SMALL};
+    }
+    p,
+    ul,
+    ol {
+        margin: 4px 0 12px;
+    }
+    ul,
+    ol {
+        padding: 0 0 0 16px;
+    }
+    li {
+        margin: 0 0 8px;
+    }
+    a {
+        color: ${props => props.theme.TYPE_GREEN};
+        &:hover {
+            text-decoration: underline;
+        }
+    }
+`;
+
+const GuideArticle = () => {
+    const [markdown, setMarkdown] = useState<string>();
+    const { article } = useSelector(state => ({
+        article: state.guide.article,
+    }));
+    const { setView } = useActions({
+        setView: guideActions.setView,
+        openArticle: guideActions.openArticle,
+    });
+
+    useEffect(() => {
+        async function loadMarkdown() {
```

Comment:
```suggestion
        const loadMarkdown = async () => {
```
To be consistent with the codebase